src/scripts/main.js

```diff
@@ -1 +1,52 @@
 'use strict';
+
+Promise((resolve, reject) => {
```

Comment:
You are missing the 'new' keyword before 'Promise'. It should be 'new Promise(...)'. The Promise object is a constructor and requires the 'new' keyword to create a new Promise instance.